package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "sanctuary-type-classes",
+  "version": "0.0.0",
+  "description": "Type classes for sanctuary-def",
```

Comment:
This is out of date. The library is now useful on its own. How about `"Standard library for Fantasy Land"`?